examples/ChatQnA/deploy/k8s_helm.md

```diff
@@ -0,0 +1,572 @@
+# Multi-node on-prem deployment with  TGI on Xeon Scalable processors on a K8s cluster using Helm Charts
+
+This deployment section covers multi-node on-prem deployment of the ChatQnA
+example with OPEA comps to deploy using the TGI service. There are several
+slice-n-dice ways to enable RAG with vectordb and LLM models, but here we will
+be covering one option of doing it for convenience : we will be showcasing  how
+to build an e2e chatQnA with Redis VectorDB and neural-chat-7b-v3-3 model,
+deployed on a Kubernetes cluster. For more information on how to setup a Xeon based Kubernetes cluster along with the development pre-requisites,
+please follow the instructions here (*** ### Kubernetes Cluster and Development Environment***). 
+For a quick introduction on Helm Charts, visit the helm section in  (**getting started**)
+
+## Overview
+
+There are several ways to setup a ChatQnA use case. Here in this tutorial, we
+will walk through how to enable the below list of microservices from OPEA
+GenAIComps to deploy a multi-node TGI megaservice solution.
+> **Note:** ChatQnA can also be deployed on a single node using Kubernetes, provided that all pods are configured to run on the same node.
```

Comment:
I'd move this note after the list of microservices so it flows better to the reader.